packages/ckeditor5-core/src/editor/editorconfig.ts

```diff
@@ -491,6 +491,10 @@ export interface EditorConfig {
 	 * 		**Note**: If {@link module:core/editor/editorconfig~EditorConfig#language `config.language`} is set to an RTL (right-to-left)
 	 * 		language, the side switches to `'left'` by default.
 	 *
+	 * 	* **`label`** (default: `'Powered by'`) &ndash; Allows changing the label displayed next to the project's logo.
+	 *
+	 * 		**Note**: Set the value to `null` to hide the label.
```

Comment:
```suggestion
	 * 		**Note**: Set the value to `null` to display the logo without a text.
```

How about the above?